sycl/doc/extensions/supported/sycl_ext_oneapi_backend_level_zero.md

```diff
@@ -388,6 +411,49 @@ make_buffer(
 Construct a SYCL buffer instance from a pointer to a Level Zero memory allocation. Please refer to <code>make_buffer</code>
 description above for semantics and restrictions.
 The additional <code>AvailableEvent</code> argument must be a valid SYCL event. The instance of the SYCL buffer class template being constructed must wait for the SYCL event parameter to signal that the memory native handle is ready to be used.
+</tr>
+
+<tr>
+<td>
+
+``` C++
+make_image(
+    const backend_input_t<backend::ext_oneapi_level_zero,
+                          image<Dimensions, AllocatorT>> &,
+    const context &Context, event AvailableEvent = {})
+```
+</td>
+<td>This API is available starting with revision 4 of this specification.
+
+Construct a SYCL image instance from a ze_image_handle_t. 
+
+Because LevelZero has no way of getting image information from an image, it must be provided.  The  <code>backend_input_t</code> is a struct type like so:
+``` C++
+struct type {
+    ze_image_handle_t ZeImageHandle;
+    sycl::image_channel_order ChanOrder;
+    sycl::image_channel_type ChanType;
+    sycl::range<Dimensions> Range;
+    ext::oneapi::level_zero::ownership Ownership{
+        ext::oneapi::level_zero::ownership::transfer};
+  };
+```
+where the Range should be ordered (width), (width, height), or (width, height, depth) for 1D, 2D and 3D images respectively, 
+with those values matching the dimensions used in the `ze_image_desc` that was used to create the `ze_image_handle_t` initially. 
+
+Example Usage
+``` C++
+sycl::backend_input_t<BE, sycl::image<2>> ImageInteropInput{ ZeHImage, ChanOrder, ChanType, ImgRange_2D, sycl::ext::oneapi::level_zero::ownership::transfer };      

```

Comment:
```suggestion
sycl::backend_input_t<BE, sycl::image<2>> ImageInteropInput{ ZeHImage, ChanOrder, ChanType, ImgRange_2D };      
```

Is this better?  Since `transfer` is the default, it doesn't need to be specified.  Or maybe you wanted to show it intentionally for emphasis?

Reply:
I don't have an opinion here.  However, that said, the ownership value is super important. If you get it wrong, you'll get crashes and/or leaks.  I used a default value because that's what we do in `make_buffer`, but if it weren't for that I don't think I'd have a default - I'd make the user specify it each time.